cookbook/rule/add_custom_action_rule_engine.rst

```diff
@@ -0,0 +1,441 @@
+How to add a custom action in the rule engine
+=============================================
+
+Quick Overview
+--------------
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
+
+This cookbook assumes that you already created a new bundle to add your custom rule. Let's assume its namespace is `Acme\\CustomBundle`.
+
+Create a custom action
+----------------------
+
+In this cookbook we are going to see how to add a custom action in the rule engine that allows to concatenate attribute value following a pattern.
+
```

Comment:
You should add a real example of what you are trying to achieve with this new rule (attribute and pattern sample) to make it more clear